### PR TITLE
Adding notes about using other versions of R and Python

### LINKED ISSUE #265: "supported" installations (python, r) is misleading
Sounds like you can't actually run other versions of R/Python. 

Instead, should have something like "CHTC-provided". 

Move text about own installation to below "CHTC-provided" heading.

### PR DESCRIPTION
**Pull request by ChristinaLK**
Closes #265

### CODE REVIEW

**Review by rachellombardi — APPROVED** [review]
Looks good!

File: uw-research-computing/r-jobs.md [hunk 1]
```diff
@@ -46,6 +48,18 @@ Supported R Installations
...
 
+If you need a newer version of R than is shown here, 
+[please contact us!](mailto:chtc@cs.wisc.edu) We want to continuously 
+add new versions of R to this list and rely on your needs to know what 
+we should add. 
+
+If you need a specific version of R not shown in this list, especially 
+if it is am older R version, we recommend using a Docker container with R installed 
+to run your jobs 
+(see CHTC's [Docker Jobs guide](/uw-research-computing/docker-jobs.html)). The
```
Comment (ChristinaLK) [inline uw-research-computing/r-jobs.md:59]:
```suggestion
(see CHTC's [Docker Jobs guide](docker-jobs.html)). The 
```

**Comment by lmichael107** [comment]
@CannonLock can you check whether the failure due to internal links (about jobs available) are concerning?

**Review by lmichael107 — APPROVED** [review]

**Comment by CannonLock** [comment]
The broken link is fixed on Master, LGTM.

**Pull request merged**